partner/partner-management-service/src/main/java/io/mosip/pms/oidc/client/service/impl/ClientManagementServiceImpl.java

```diff
@@ -182,7 +190,9 @@ private static List<String> getReqAttributeFromPolicyJson(JSONObject policyObjec
 				throw new PartnerServiceException(ErrorCode.POLICY_HAVING_MANDATORY_AUTHS.getErrorCode(),
 						ErrorCode.POLICY_HAVING_MANDATORY_AUTHS.getErrorMessage());
 			}
-			attributes.add(key);
+			if(!attributes.contains(key)) {
```

Comment:
Instead of this can we make "attributes" as Set

Reply:
done anusha